### PR TITLE
added a column to Act.cvs

### PR DESCRIPTION
**Pull request by mira-gruenwald**
to record whether  an act is fictional or not.
see #366

added a new line to the data dict.

### CODE REVIEW

File: data/helper/data_dictionary.csv [hunk 1]
```diff
@@ -202,6 +205,7 @@ Act,substantial_discussion,substantial comment or quotation from source, 'unknow
 Act,source_id,Evidence for activity,,2017-07-04,LH,2018-02-01,DP
 Act,page_number,Page number in text that the column source_id refers to,,2017-07-04,LH,2017-07-04,LH
 Act,source_status,Genre_type of the source. '01' for informational works / '02' for literary non-fiction / '03' for fiction,01: http://id.loc.gov/authorities/genreForms/gf2014026113.html / 02: http://id.loc.gov/authorities/genreForms/gf2014026074.html / 03: http://id.loc.gov/authorities/genreForms/gf2014026339.html,2018-01-31,MG,2018-03-08,MG
+Act,act_status,Add F to state the fictionality of an act. If the agent of an act is fictional (F) the Act is fictional by default. If the agent is based on a real-life character the act_status is determined by the literary_status of the source.,,2019-04-09,MG,2019-04-09,MG
```
Comment (duncdrum) [inline data/helper/data_dictionary.csv]:
```suggestion
Act,act_status,Add F to mark fictional reading acts. By definition if either Act.agent, or Act.act_object have a fictional character_status or literary_status (F) property, the resulting Act.act_status is fictional as well. Act.source_status property is insufficient to assert the fictionality of an act. Where real-world personas appear in fictional texts, contributors must indicate evidence for the act_status in Act.commentary,,2019-04-09,MG,2019-04-09,MG
```

**Comment by LenaHenningsen** [comment]
It took me a bit to get into the act.agent dictum, but seen fine by me. What do you think Mira?

 

Von: Duncan Paterson <notifications@github.com>
Antworten an: readchina/ReadingData <reply@reply.github.com>
Datum: Dienstag, 9. April 2019 um 11:05
An: readchina/ReadingData <ReadingData@noreply.github.com>
Cc: LenaHenningsen <Lena.Henningsen@sinologie.uni-freiburg.de>, Review requested <review_requested@noreply.github.com>
Betreff: Re: [readchina/ReadingData] added a column to Act.cvs (#370)

 

@duncdrum requested changes on this pull request.

I tried to make the act definition a bit less ambigous, please have a look and discuss

In data/helper/data_dictionary.csv:
> @@ -202,6 +205,7 @@ Act,substantial_discussion,substantial comment or quotation from source, 'unknow
 
 Act,source_id,Evidence for activity,,2017-07-04,LH,2018-02-01,DP
 
 Act,page_number,Page number in text that the column source_id refers to,,2017-07-04,LH,2017-07-04,LH
 
 Act,source_status,Genre_type of the source. '01' for informational works / '02' for literary non-fiction / '03' for fiction,01: http://id.loc.gov/authorities/genreForms/gf2014026113.html / 02: http://id.loc.gov/authorities/genreForms/gf2014026074.html / 03: http://id.loc.gov/authorities/genreForms/gf2014026339.html,2018-01-31,MG,2018-03-08,MG
 
+Act,act_status,Add F to state the fictionality of an act. If the agent of an act is fictional (F) the Act is fictional by default. If the agent is based on a real-life character the act_status is determined by the literary_status of the source.,,2019-04-09,MG,2019-04-09,MG
 
⬇️ Suggested change 
-Act,act_status,Add F to state the fictionality of an act. If the agent of an act is fictional (F) the Act is fictional by default. If the agent is based on a real-life character the act_status is determined by the literary_status of the source.,,2019-04-09,MG,2019-04-09,MG
 
+Act,act_status,Add F to mark fictional reading acts. By definition if either Act.agent, or Act.act_object have a fictional character_status or literary_status (F) property, the resulting Act.act_status is fictional as well. Act.source_status property is insufficient to assert the fictionality of an act. Where real-world personas appear in fictional texts, contributors must indicate evidence for the act_status in Act.commentary,,2019-04-09,MG,2019-04-09,MG

File: data/helper/data_dictionary.csv (hunk 1 above)
Comment (mira-gruenwald) [reply data/helper/data_dictionary.csv]: ok, thanks! This definition is more precise.

**Comment by mira-gruenwald** [comment]
Hello @LenaHenningsen, I committed the suggested changes by Duncan. Do you agree now with this PR? In case you do, you can merge the branch into yours.

**Pull request merged**